src/main/java/dev/ebrydeu/spring_boot_library/repositrories/MessageRepository.java

```diff
@@ -0,0 +1,30 @@
+package dev.ebrydeu.spring_boot_library.repositrories;
+
+import dev.ebrydeu.spring_boot_library.domain.entities.Message;
+import org.springframework.data.repository.ListCrudRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.jpa.repository.Modifying;
+import jakarta.transaction.Transactional;
+import java.util.List;
+
+public interface MessageRepository extends ListCrudRepository<Message, Long> {
+    List<Message> findAll();
```

Comment:
Remove it. `CrudRepository` interface already have this method no point to override